README.md

```diff
@@ -2,23 +2,72 @@
 
 # [ghuser.io](https://github.com/ghuser-io/ghuser.io)'s database
 
-It is here to cache GitHub's data. During the prototyping phrase, it's just a set of
-[JSON files](data/) that we update once per day by running [this bot](fetchBot/) on an
-[EC2 instance](https://github.com/ghuser-io/ghuser.io/blob/master/aws/ec2).
+This repository:
+* Serves as database for the [ghuser.io](https://github.com/ghuser-io/ghuser.io) Reframe app.\
+The DB consists of the [JSON files in `data`](data/).
+* Provides scripts to update the database. 
+
+In particular, [FetchBot](fetchBot/) runs daily on an [EC2 instance](https://github.com/ghuser-io/ghuser.io/blob/master/aws/ec2).
 
 ## Table of Contents
 
 <!-- toc -->
 
-- [Updating the database](#updating-the-database)
-- [Serving one more user](#serving-one-more-user)
-- [Deleting a profile](#deleting-a-profile)
+- [Setup](#setup)
+- [Usage](#usage)
+- [Implementation](#implementation)
 
 <!-- tocstop -->
 
-## Updating the database
+## Setup
 
-Several scripts form a pipeline for updating the database. Here is the data flow:
```

Comment:
I think it's quite relavant to state that this is kind of a pipeline and there is a flow. For example, seeing this data flow, you can understand that users/*.json file can be updated in parallel (which is important since it takes hours), and you understand that you can't start updating the repos while the users are in an inconsistent state. Also, you see that updating the orgs and updating the repos can be done in parallel.

Reply:
Okay, restoring. I didn't understand how specific were those keywords. Is there a name for this kind of diagrams?

Reply:
it's not standard, but it's kind of a data-flow-diagram I'd say